rpcs3/util/serialization_ext.cpp

```diff
@@ -659,7 +659,7 @@ void compressed_serialization_file_handler::file_writer_thread_op()
 	compressed_stream_data& stream = *m_stream;
 
 	// Data recheck after an abort request is detected so there will not be any missed data
-	bool rechecked = false;
+	[[maybe_unused]] bool rechecked = false;
```

Comment:
This should either be removed entirely or added where it was supposed to

Reply:
Yeah it can be removed.

Reply:
removed